Bookcase/Bookcase.csproj

```diff
@@ -72,6 +74,7 @@
   <ItemGroup>
     <Analyzer Include="..\packages\Pathoschild.Stardew.ModBuildConfig.2.1.0\analyzers\dotnet\cs\StardewModdingAPI.ModBuildConfig.Analyzer.dll" />
   </ItemGroup>
+  <ItemGroup />
```

Comment:
Is this valid XML?

Reply:
open close tag
<tag/>
But yeah idk why that got added, it's basically a blank itemgroup there.